internal/cmd/gocdk/init_test.go

```diff
@@ -147,6 +148,83 @@ func TestInit(t *testing.T) {
 		}
 	})
 }
+func TestInferModulePath(t *testing.T) {
```

Comment:
Since most of the content of each test is the same, this is a good opportunity to use a table-driven test.

https://blog.golang.org/subtests